bpmn-model/src/main/java/io/zeebe/model/bpmn/util/time/Timer.java

```diff
@@ -0,0 +1,10 @@
+package io.zeebe.model.bpmn.util.time;
+
+public interface Timer {
+
+  Interval getInterval();
+
+  int getRepetitions();
+
+  long getDueDate();
```

Comment:
If `getDueDate` takes in the current epoch in milli (or rather, `startEpoch` as a long), then we can get rid of the dependency to `zb-util` which I assume is only there for the `ActorClock` stuff? Then the `TimeDate` impl can simply ignore the given `startEpoch`, just make sure it's documented.